src/extension.ts

```diff
@@ -320,12 +320,36 @@ class GitBackupSync {
 			return false;
 		}
 
+		let backupBranchName: string;
+		let branchInfo = await this._branchInfo.get(this._config.branchInfoPath, currentBranchName);
+		if (branchInfo === undefined) {
```

Comment:
I think it's not just when the branch info file isn't present; there's also the case where the branch info file doesn't contain the newest information. So a branch info file may be present, but the branch might not be specified in the branch info file

Reply:
Sorry, I don't understand the problem here lol. I think as long as the current branch information does not exist in the branch info file, the value of branchInfo will be undefined. Are there any other possibilities?

Reply:
oops sry nvm, misread